src/Compilers/VisualBasic/Portable/Compilation/VisualBasicCompilation.vb

```diff
@@ -2724,7 +2728,36 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 Throw New ArgumentException(VBResources.NoNoneSearchCriteria, NameOf(filter))
             End If
 
-            Return New SymbolSearcher(Me).GetSymbolsWithName(predicate, filter, cancellationToken)
+            Return New PredicateSymbolSearcher(Me, filter, predicate, cancellationToken).GetSymbolsWithName()
+        End Function
+
+        ''' <summary>
+        ''' Return true if there Is a source declaration symbol name that matches the provided name.
```

Comment:
>Is [](start = 33, length = 2)

Typo: `Is` from copy&paste (in a couple of places)

Reply:
#fixed.  thanks!